test/src/tests/generated/body-complex/polymorphism.go

```diff
@@ -189,6 +189,61 @@ func (client PolymorphismClient) PutComplicatedResponder(resp *http.Response) (r
 	return
 }
 
+// PutMissingDiscriminator put complex types that are polymorphic, omitting the discriminator
+//
+func (client PolymorphismClient) PutMissingDiscriminator(ctx context.Context, complexBody BasicSalmon) (result SalmonModel, err error) {
```

Comment:
>PutMissingDiscriminator [](start = 33, length = 23)

Arent there new tests that6 use this func?

Reply:
Yes I don't think we have for marshaling when the discriminator is missing.